asllib/AST.mli

```diff
@@ -185,8 +188,16 @@ and int_constraint =
   | Constraint_Range of (expr * expr)
       (** In the range of these two statically evaluable values.*)
 
-and int_constraints = int_constraint list
-(** The int_constraints represent the union of the individual constraints.*)
+(** The int_constraints constraints the integer type to a certain subset.*)
+and int_constraints =
+  | UnConstrained  (** The normal, unconstrained, integer type. *)
```

Comment:
nit: unconstrained is a single word so perpahps use Unconstrained as a constructor?